### ISSUE TITLE
Application window doesn't open after packaging

### ISSUE THREAD

**Issue by melramkhelawan** [ElectronNET/Electron.NET#387]
* **Version**: 7.30.2

.NET Core 3.1

* **Target**: Windows

The command prompt window opens but the application doesn't. 

Steps to Reproduce:

1. Build using electronize build /target win
2. Run the exe

**Comment by GregorBiswanger** [comment]
Can you provide me your application with source code if necessary? 

The current version works without problems for us.

**Comment by ipax77** [comment]
Some users of my app do have the same problem – it is stuck at:
```
Electron Socket IO Port: 8000
Electron Socket started on port 8000 at 127.0.0.1
ASP.NET Core Port: 8001
```

On my Desktop and a virtual machine  it is working fine, but I was able to repoduce it on my laptop (all (Win10 64)

The code is available here [here](https://github.com/ipax77/sc2dsstats/tree/master/sc2dsstats.desktop)
but I have the same result (desktop/virtual machine ok, laptop nok) with a clean asp dotnet core blazor app using
ElectronNET.API & ElectronNET.CLI Version 8.31.1
electron-builder Version 22.6.0 using electronize build /target win
dotnet Version3.1.201

I reverted back to ElectronNET.API & ElectronNET.CLI Version 7.30.2 and it is working as expected on my laptop.

**Comment by drozdzamadeusz** [comment]
Any updated on this issue? A have exactly the same problem, after update program starts on my computer but not on any other.

**Comment by GregorBiswanger** [comment]
@ipax77 I try to start your project with VS2019 and get an error:
![image](https://user-images.githubusercontent.com/7336300/81437416-6c234380-916b-11ea-933a-39b9da07e2b7.png)

@drozdzamadeusz can you provide me with your project or can you reproduce it please?

**Comment by GregorBiswanger** [comment]
@ipax77 FYI
![image](https://user-images.githubusercontent.com/7336300/81437549-a1c82c80-916b-11ea-9c05-8104b863dc43.png)

**Comment by ipax77** [comment]
@GregorBiswanger fixed the sln file, sry.
I was able to reproduce it with the clean sample dotnet core blazor project, too (with counter and weatherforcast).

**Comment by drozdzamadeusz** [comment]
@GregorBiswanger yes of course, here is source of my project: https://github.com/drozdzamadeusz/talker-app

**Comment by drozdzamadeusz** [comment]
@GregorBiswanger I figured out that to run application every user has to have NET Core 3.1 SDK installed. I have no idea why this situation occurs, before update to 8.31.1 application started normally on computers without NET SDK installed.

**Comment by GregorBiswanger** [comment]
@ipax77 I think @drozdzamadeusz is right here.. it is no longer produced as self contained.. I can't say where that comes from... that must have slipped in with a pull request.. I will change and test it locally.. if then everything works, it will be released immediately for update 8.31.2.

![image](https://user-images.githubusercontent.com/7336300/81482573-4fa10d00-9238-11ea-9753-7f9db38a69d7.png)

**Comment by GregorBiswanger** [comment]
Okay, will be fixed with Electron.NET 8.31.2...

![image](https://user-images.githubusercontent.com/7336300/81484357-25098100-9245-11ea-84e3-6273fb978723.png)

**Comment by GregorBiswanger** [comment]
Fixed and will be available with update 8.31.2.
